libs/testserver/fake_workspace.go

```diff
@@ -170,6 +170,12 @@ func (s *FakeWorkspace) CurrentUser() iam.User {
 
 func (s *FakeWorkspace) WorkspaceGetStatus(path string) Response {
 	defer s.LockUnlock()()
+	if !strings.HasPrefix(path, "/") {
+		return Response{
+			StatusCode: 500,
+			Body:       map[string]string{"message": fmt.Sprintf("Path (%s) doesn't start with '/'", path)},
```

Comment:
q - Is this something that you used to reproduce this but it did not make it in final test? I don't see this message in test output.

Reply:
Makes sense but it's unrelated. At the very least should be mentioned in the PR description (ideally separate PR).

Reply:
Removed it from this PR https://github.com/databricks/cli/pull/3319/commits/975797f930177ce4e07e45d9d309f8a5fd542ab8